doc/content/reference/configuration/the-things-stack.md

```diff
@@ -0,0 +1,174 @@
+
+- `tls.root-ca`: Location of TLS root CA certificate (optional)
+- `tls.insecure-skip-verify`: Skip verification of certificate chains (insecure)
+
+## gRPC Options
+
+The `grpc` options configure how The Things Stack listens for gRPC connections. The format is `host:port`. When listening on TLS ports, it uses the global [TLS configuration]({{< ref "#tls-options" >}}).
+
+- `grpc.listen`: Address for the TCP gRPC server to listen on
+- `grpc.listen-tls`: Address for the TLS gRPC server to listen on
+
+When running a cluster in a trusted network, you can allow sending credentials over insecure connections with the `allow-insecure-for-credentials` option:
+
+- `grpc.allow-insecure-for-credentials`: Allow transmission of credentials over insecure transport
+
+## HTTP Options
+
+The `http` options configure how The Things Stack listens for HTTP connections. The format is `host:port`. When listening on TLS ports, it uses the global [TLS configuration]({{< ref "#tls-options" >}}).
+
+- `http.listen`: Address for the HTTP server to listen on
+- `http.listen-tls`: Address for the HTTPS server to listen on
+
+The Things Stack uses secure cookies that are encrypted with a `block-key` and signed with a `hash-key`. In production deployments you'll want these to stay the same between restarts. The keys are encoded as hex.
+
+- `http.cookie.block-key`: Key for cookie contents encryption (16, 24 or 32 bytes)
+- `http.cookie.hash-key`: Key for cookie contents verification (32 or 64 bytes)
+
+The Things Stack serves a number of internal endpoints for health, metrics and debugging. These will usually be disabled or password protected in production deployments.
+
+- `http.health.enable`: Enable health check endpoint on HTTP server
+- `http.health.password`: Password to protect health endpoint (username is health)
+- `http.metrics.enable`: Enable metrics endpoint on HTTP server
+- `http.metrics.password`: Password to protect metrics endpoint (username is metrics)
+- `http.pprof.enable`: Enable pprof endpoint on HTTP server
+- `http.pprof.password`: Password to protect pprof endpoint (username is pprof)
+
+It is possible to redirect users to the canonical URL of a deployment. There are options to redirect to a given host, or redirect from HTTP to HTTPS.
+
+- `http.redirect-to-host`: Redirect all requests to one host
+- `http.redirect-to-tls`: Redirect HTTP requests to HTTPS
+
+The HTTP server serves static files for the web UI. If these files are not in the standard location, you may need to change the search path.
+
+- `http.static.mount`: Path on the server where static assets will be served
+- `http.static.search-path`: List of paths for finding the directory to serve static assets from
+
+## Interop Options
+
+The Things Stack supports interoperability with the LoRaWAN Backend Interfaces specification. The following options are used to configure the server for this.
```

Comment:
```suggestion
The Things Stack supports interoperability according to LoRaWAN Backend Interfaces specification. The following options are used to configure the server for this.
```

Reply:
IMO this section is a bit confusing, since interoperability is also configured using `{as,ns}.interop.*`. I think we should specify explicitly here that this only concerns the server-side of things and clients are configured per-component

Reply:
Possibly with a reference to https://github.com/TheThingsNetwork/lorawan-stack/pull/1266

Reply:
Let's merge this as-is and then update it in #1266